src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -1466,6 +1466,19 @@ public new string DisplayName
         }
         internal string displayName = null;
 
+        /// <summary>
+        /// Account under which the service should run
+        /// </summary>
+        /// <value></value>
+        [Parameter]
+        [Credential()]
+        public PSCredential Credential
+        {
+            get { return credential; }
+            set { credential = value; }
+        }
+        internal PSCredential credential = null;
```

Comment:
For the new code, it is preferable to use auto property. Especially because `credential` is not used.